app/components/participant_status_tag_component.rb

```diff
@@ -35,6 +36,10 @@ def tag_attributes
     { text: "Contacting for information", colour: "grey" }
   end
 
+  def training_status_withdrawn?
+    (induction_record || profile).training_status_withdrawn?
```

Comment:
induction_record seems to be used with `&` operator almost everywhere. Should it be used here as well to make it nil safe?

Reply:
If induction_record is nil here it will default to using the profile